src/state-item.c

```diff
@@ -0,0 +1,417 @@
+//
+//  state-item.c
+//  bison
+//
+//  Created by Vincent Imbimbo on 10/15/19.
+//  Copyright © 2019 Vincent Imbimbo. All rights reserved.
+//
+
```

Comment:
Please, rewrite all your headers so that there are exactly the same as the others in Bison.

``` C
/* <DESCRIPTION OF THIS FILE>

   Copyright (C) 2020 Free Software Foundation, Inc.

   This file is part of Bison, the GNU Compiler Compiler.

   This program is free software: you can redistribute it and/or modify
   it under the terms of the GNU General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License
   along with this program.  If not, see <http://www.gnu.org/licenses/>.  */
```

You can add below this something like

``` C
/* Written by Vincent Imbimbo.  */
```